sdk/core/core-client-rest/src/sendRequest.ts

```diff
@@ -31,65 +30,29 @@ export async function sendRequest(
   method: HttpMethods,
   url: string,
   pipeline: Pipeline,
-  options: RequestParameters = {},
+  options: InternalRequestParameters = {},
   customHttpClient?: HttpClient
 ): Promise<HttpResponse> {
   const httpClient = customHttpClient ?? getCachedDefaultHttpsClient();
   const request = buildPipelineRequest(method, url, options);
-
   const response = await pipeline.sendRequest(httpClient, request);
-
-  const rawHeaders: RawHttpHeaders = response.headers.toJSON();
-
-  const parsedBody: RequestBodyType | undefined = getResponseBody(response);
+  const headers = response.headers.toJSON();
+  const stream = response.readableStreamBody ?? response.browserStreamBody;
+  const parsedBody = getResponseBody(response);
```

Comment:
I think we can avoid calling `getResponseBody` in the stream case and leave it as undefined, right?

Reply:
Good point, it is addressed in https://github.com/Azure/azure-sdk-for-js/pull/28111/commits/e68a10fff42350eaeecc23123d7060c789068f18.